en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,65 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In Open Orchestra the ``Content`` are composed of a set of base attributes and a collection of
+``ContentAttribute`` (See `content`_ documentation for more information). The ``ContentAttribute``
+are configured by the `content type`_.
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+
+Sometimes, there is no easy way to render the value to the backoffice user.
+A custom ``ContentAttribute`` could be store as an array, which cannot be displayed directly in the dataTables plugin.
+In that way the ``ContentAttribute`` class provides a specific attribute ``stringValue`` which is the
+representation of the ``value`` in an HTML string.
+
+Transformers
+------------
+
+Open Orchestra uses a ``ValueTransformer`` strategy manager to generate the HTML strings for
+multiple ``value`` types. Open Orchestra already provides some strategies located in
+``OpenOrchestra\Backoffice\ValueTransformer\Strategies``.
+
+Strategy creation
+~~~~~~~~~~~~~~~~~
+
+Service used as ``ValueTransformer`` must be tagged as ``open_orchestra_backoffice.value_transformer.strategy``.
+
+.. code-block:: yaml
+
+    tags:
+        - { name: open_orchestra_backoffice.value_transformer.strategy }
+
+Such services also need to implement ``OpenOrchestra\Backoffice\ValueTransformer\ValueTransformerInterface``.
```

Comment:
I think it would be useful to detail the implementation of the interface, as there is no documentation in the interface itself.
Also, I have some difficulty to grasp the use cases here : if i have a content with an array attribute containing mongo ids, how should I implement the interface to get it display a field of the linked mongo documents ? Maybe a concrete example would help.